src/app/asset-page/asset-page.component.ts

```diff
@@ -902,6 +902,20 @@ export class AssetPage implements OnInit, OnDestroy {
     return false
   }
 
+  private getReasonForAuth() {
+    if(this.showAccessDeniedModal) {
+      return "not_authorized"
+    } else {
+      if(this.assetGroupId) {
+        return "image_group"
```

Comment:
The `image_group` and `encrypted_access` strings are just placeholders for now until we can chat with data platform.